packages/manager/core/generator/app/templates/package.json.hbs

```diff
@@ -40,10 +41,16 @@
     "tailwindcss": "^3.4.4"
   },
   "devDependencies": {
-    "@cucumber/cucumber": "^10.3.1",
+    "@testing-library/dom": "^9.3.4",
+    "@testing-library/jest-dom": "^6.4.2",
+    "@testing-library/react": "^14.2.1",
+    "@tanstack/react-query-devtools": "^5.51.21",
     "@ovh-ux/manager-vite-config": "*",
+    "@vitest/coverage-v8": "^2.1.2",
     "typescript": "^5.1.6",
-    "vite": "^5.2.13"
+    "vite": "^5.2.13",
+    "vitest": "^2.1.2"
```

Comment:
Think to check if the project can build with this version. I tried before and problem with the general `yarn.lock` allow to test the uApp but not the full project.

Maybe you should add 2.1.3 as hycu

Reply:
on my side, I can launch the project, but I had to 'yarn install' at the root of the manager to be able to launch the new generated uapp project

In master, we have  2.1.2 for hycu. There is no project that use 2.1.3 at the moment